dnf/transaction_sr.py

```diff
@@ -57,21 +57,19 @@ def __init__(self, filename, errors):
 
```

Comment:
I would like to at least rename the `TransactionFileError` to something that doesn't contain "File", as it no longer has to be related to a file. I'm thinking e.g. `TransactionReplayError`. The only issue I have here is that it's actually not that clear what's the difference to `TransactionError`, but maybe it's ok. Maybe adding short docstrings to both classes describing their relation would be helpful and enough to clear the potential confusion?

Reply:
`TransactionReplayError` makes sense to me. Changed.